core/controllers/base.py

```diff
@@ -196,6 +196,11 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
             auth_services.destroy_auth_session(self.response)
             self.redirect(user_services.create_login_url(self.request.uri))
             return
+        except auth_domain.UserDisabledError:
```

Comment:
Just to double-check, was this a consequence of the lib upgrades?

Reply:
Yes.

Reply:
Could you please explain more?

Reply:
Basically, the previous versions returned `None` in the case when the user was disabled. Now the library throws and error when the user is disabled, we pass the error down the line. Because we use disable only for users to be deleted we redirect them to pending deletion page.